### ISSUE TITLE
Run cell says "ImportError: DLL load failed: The specified path is invalid"

### ISSUE THREAD

**Issue by Scoodood** [DonJayamanne/vscodeJupyter#80]
### Environment data
VS Code version: 1.61
Jupyter Extension version: 1.1.3
OS and version: Win7 Ultimate x64 SP1

### Logs
Output from ```Jupyter``` output panel
```
ImportError                               Traceback (most recent call last)
~\Anaconda3\lib\site-packages\numpy\core\__init__.py in <module>()
     15 try:
---> 16     from . import multiarray
     17 except ImportError as exc:

ImportError: DLL load failed: The specified path is invalid.

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-2-620aef27cc09> in <module>()
      1 #%%
----> 2 import matplotlib.pyplot as plt
      3 import numpy as np
      4
      5 x = np.linspace(0, 20, 100)

~\Anaconda3\lib\site-packages\matplotlib\__init__.py in <module>()
    120 # cbook must import matplotlib only within function
    121 # definitions, so it is safe to import from it here.
--> 122 from matplotlib.cbook import is_string_like, mplDeprecation, dedent, get_label
    123 from matplotlib.compat import subprocess
    124 from matplotlib.rcsetup import (defaultParams,

~\Anaconda3\lib\site-packages\matplotlib\cbook.py in <module>()
     30 from weakref import ref, WeakKeyDictionary
     31
---> 32 import numpy as np
     33 import numpy.ma as ma
     34

~\Anaconda3\lib\site-packages\numpy\__init__.py in <module>()
    140         return loader(*packages, **options)
    141
--> 142     from . import add_newdocs
    143     __all__ = ['add_newdocs',
    144                'ModuleDeprecationWarning',

~\Anaconda3\lib\site-packages\numpy\add_newdocs.py in <module>()
     11 from __future__ import division, absolute_import, print_function
     12
---> 13 from numpy.lib import add_newdoc
     14
     15 ###############################################################################

~\Anaconda3\lib\site-packages\numpy\lib\__init__.py in <module>()
      6 from numpy.version import version as __version__
      7
----> 8 from .type_check import *
      9 from .index_tricks import *
     10 from .function_base import *

~\Anaconda3\lib\site-packages\numpy\lib\type_check.py in <module>()
      9            'common_type']
     10
---> 11 import numpy.core.numeric as _nx
     12 from numpy.core.numeric import asarray, asanyarray, array, isnan, zeros
     13 from .ufunclike import isneginf, isposinf

~\Anaconda3\lib\site-packages\numpy\core\__init__.py in <module>()
     24 Original error was: %s
     25 """ % (exc,)
---> 26     raise ImportError(msg)
     27
     28 for envkey in env_added:

ImportError:
Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
If you're working with a numpy git repo, try `git clean -xdf` (removes all
files not under version control).  Otherwise reinstall numpy.

Original error was: DLL load failed: The specified path is invalid.
```

Output from ```Console window``` (Help->Developer Tools menu)
```
[File Watcher] System.ComponentModel.Win32Exception (0x80004005): Incorrect function
extensionService.ts:409 [C:\Users\JC\.vscode\extensions\donjayamanne.python-0.7.0]: Command `jupyter.runSelectionLine` appears multiple times in the `commands` section.
extensionService.ts:409 [C:\Users\JC\.vscode\extensions\donjayamanne.python-0.7.0]: Command `jupyter.execCurrentCell` appears multiple times in the `commands` section.
extensionService.ts:409 [C:\Users\JC\.vscode\extensions\donjayamanne.python-0.7.0]: Command `jupyter.execCurrentCellAndAdvance` appears multiple times in the `commands` section.
extensionService.ts:409 [C:\Users\JC\.vscode\extensions\donjayamanne.python-0.7.0]: Command `jupyter.gotToPreviousCell` appears multiple times in the `commands` section.
extensionService.ts:409 [C:\Users\JC\.vscode\extensions\donjayamanne.python-0.7.0]: Command `jupyter.gotToNextCell` appears multiple times in the `commands` section.
mime.ts:59 Overwriting extension <<.html>> to now point to mime <<text/x-html>>
(anonymous) @ mime.ts:59
t.registerTextMime @ mime.ts:53
e._mergeLanguage @ languagesRegistry.ts:127
e._registerLanguage @ languagesRegistry.ts:103
e._registerLanguages @ languagesRegistry.ts:59
(anonymous) @ languagesRegistry.ts:49
e.invoke @ callbackList.ts:63
e.fire @ event.ts:123
e.registerLanguages @ modesRegistry.ts:38
(anonymous) @ workbenchModeService.ts:122
e._handle @ extensionsRegistry.ts:115
e.acceptUsers @ extensionsRegistry.ts:99
e._handleExtensionPoint @ extensionService.ts:392
(anonymous) @ extensionService.ts:313
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
l @ winjs.base.raw.js:1813
n.Class.derive.join.r.forEach.$.then.o.(anonymous function) @ winjs.base.raw.js:1838
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
l @ winjs.base.raw.js:1813
n.Class.derive.join.r.forEach.$.then.o.(anonymous function) @ winjs.base.raw.js:1838
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
then @ winjs.base.raw.js:1566
enter @ winjs.base.raw.js:747
_run @ winjs.base.raw.js:1068
_setCompleteValue @ winjs.base.raw.js:1052
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
mime.ts:59 Overwriting extension <<.htm>> to now point to mime <<text/x-html>>
(anonymous) @ mime.ts:59
t.registerTextMime @ mime.ts:53
e._mergeLanguage @ languagesRegistry.ts:127
e._registerLanguage @ languagesRegistry.ts:103
e._registerLanguages @ languagesRegistry.ts:59
(anonymous) @ languagesRegistry.ts:49
e.invoke @ callbackList.ts:63
e.fire @ event.ts:123
e.registerLanguages @ modesRegistry.ts:38
(anonymous) @ workbenchModeService.ts:122
e._handle @ extensionsRegistry.ts:115
e.acceptUsers @ extensionsRegistry.ts:99
e._handleExtensionPoint @ extensionService.ts:392
(anonymous) @ extensionService.ts:313
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
l @ winjs.base.raw.js:1813
n.Class.derive.join.r.forEach.$.then.o.(anonymous function) @ winjs.base.raw.js:1838
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
l @ winjs.base.raw.js:1813
n.Class.derive.join.r.forEach.$.then.o.(anonymous function) @ winjs.base.raw.js:1838
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
then @ winjs.base.raw.js:1566
enter @ winjs.base.raw.js:747
_run @ winjs.base.raw.js:1068
_setCompleteValue @ winjs.base.raw.js:1052
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-html-1.2.0]: The "Lorem"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-html-1.2.0]: The "URL"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-html-1.2.0]: The "With"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "Model"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "Model Admin"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "CharField"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "Foreign Key"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "Override Save Method"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "View"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "ListView"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "DetailView"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionService.ts:407 [C:\Users\JC\.vscode\extensions\bibhasdn.django-snippets-1.1.0]: The "get_context_data"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.
e._logMessageInConsole @ extensionService.ts:407
e._handleExtensionPointMessage @ extensionService.ts:331
(anonymous) @ extensionService.ts:310
e._msg @ extensionsRegistry.ts:35
e.warn @ extensionsRegistry.ts:49
a @ TMSnippets.ts:137
(anonymous) @ TMSnippets.ts:150
p @ TMSnippets.ts:147
(anonymous) @ TMSnippets.ts:95
g @ winjs.base.raw.js:1170
enter @ winjs.base.raw.js:852
_run @ winjs.base.raw.js:1068
_completed @ winjs.base.raw.js:1036
(anonymous) @ async.ts:643
(anonymous) @ C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\graceful-fs\graceful-fs.js:78
readFileAfterClose @ original-fs.js:439
extensionHost.ts:384 [Extension Host] Starting websocket ws://localhost:8890/api/kernels/b1c0ca68-dd7f-4b21-bd5a-81c8c8d8456a
extensionHost.ts:384 [Extension Host] Timeout
webview.ts:151 [Embedded Page] reloaded results window at time 1507153248836ms
3webview.ts:151 [Embedded Page] [object Object]
```

### Steps to reproduce:
I have virtualbox Win7x64 VM and I use the following step to reproduce the problem.

1. Uninstall Anaconda (4.3) and reboot
2. Reinstall Anaconda 5.0.0 with Python 3.6.2 x64
3. Uninstall Python extension within vscode
4. Reinstall Python extension Pack and reload vscode
4. Click on "run cell" the following code
```
#%%
import matplotlib.pyplot as plt
import numpy as np
x = np.linspace(0, 20, 100)
plt.plot(x, np.sin(x))
plt.show()
```

#### Then I launch `ipython` from the integrated terminal within vscode, and tried to import the `numpy` library I also get the same error.
```
In [1]: import numpy as np
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
~\Anaconda3\lib\site-packages\numpy\core\__init__.py in <module>()
     15 try:
---> 16     from . import multiarray
     17 except ImportError as exc:

ImportError: DLL load failed: The specified path is invalid.

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-1-4ee716103900> in <module>()
----> 1 import numpy as np

~\Anaconda3\lib\site-packages\numpy\__init__.py in <module>()
    140         return loader(*packages, **options)
    141
--> 142     from . import add_newdocs
    143     __all__ = ['add_newdocs',
    144                'ModuleDeprecationWarning',

~\Anaconda3\lib\site-packages\numpy\add_newdocs.py in <module>()
     11 from __future__ import division, absolute_import, print_function
     12
---> 13 from numpy.lib import add_newdoc
     14
     15 ###############################################################################

~\Anaconda3\lib\site-packages\numpy\lib\__init__.py in <module>()
      6 from numpy.version import version as __version__
      7
----> 8 from .type_check import *
      9 from .index_tricks import *
     10 from .function_base import *

~\Anaconda3\lib\site-packages\numpy\lib\type_check.py in <module>()
      9            'common_type']
     10
---> 11 import numpy.core.numeric as _nx
     12 from numpy.core.numeric import asarray, asanyarray, array, isnan, zeros
     13 from .ufunclike import isneginf, isposinf

~\Anaconda3\lib\site-packages\numpy\core\__init__.py in <module>()
     24 Original error was: %s
     25 """ % (exc,)
---> 26     raise ImportError(msg)
     27
     28 for envkey in env_added:

ImportError:
Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
If you're working with a numpy git repo, try `git clean -xdf` (removes all
files not under version control).  Otherwise reinstall numpy.

Original error was: DLL load failed: The specified path is invalid.
```

#### To confirm the problem only affect vscode, I tried to do the same on Win7 command prompt, launched `ipython` and import the numpy library. But it run fine. This confirm the issue affect the vscode.
I had tried both vscode x86 and x64, so far not luck.

![capture](https://user-images.githubusercontent.com/15989924/31201858-1e0ea392-a915-11e7-8752-eb762a8a87f9.PNG)

**Comment by gandhis1** [comment]
Do you have any other installations of Python that are not 64-bit?

**Comment by DonJayamanne** [comment]
Can you try the command prompt in your VM, i.e. forget vs code, just try in a terminal in side your VM and then try the command.

**Comment by Scoodood** [comment]
Hi Don,

Here are the tests I run so far. Tomorrow I am going to repeat Test Scenario #2 again.

Steps:
* uninstall existing Anaconda Distribution
* reinstall the latest Anaconda Distribution with Python 3.6.2 (x86 and x64)
* tried VScode 1.6.1 (x86 and x64)
* uninstall all python related extension within VScode
* reinstall Python Extension Pack in VScode

How to test:
* run the code on IPython shell on terminal (outside vscode) 
* run the code on Jupiter notebook cell
* run the code on Sublime Text 3
* run the code on VScode integrated terminal
* run the code on VScode by clicking the %%run cell

Result:
OK - if the code run properly in all scenario


Test Scenario
1. Win7x64 host (Active Directory networked computer)
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x64 + F-secure enabled* -> OK
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x64 + F-secure enabled* -> OK
*antivirus program

2. Win7x64 VM (within Win7x64 host)
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee enabled* -> numpy failed to load**
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee enabled* -> numpy failed to load**
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee enabled* -> numpy failed to load**
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee enabled* -> numpy failed to load**
*during installation of Anaconda
**failed to load in VScode, while other are fine

3. OSX 10.12.6 host (standalone computer)
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x64 -> OK

4. Win7x64 VM (within OSX 10.12.6 host)
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee enabled* -> OK
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee enabled* -> OK
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee disabled* -> OK
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x64 + McAfee disabled* -> OK
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x86 + McAfee enabled* -> OK
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x86 + McAfee enabled* -> OK
VScode 1.6.1 x86 + Anaconda 5.0.0 + Python 3.6.2 x86 + McAfee disabled* -> OK
VScode 1.6.1 x64 + Anaconda 5.0.0 + Python 3.6.2 x86 + McAfee disabled* -> OK
*during installation of Anaconda

**Comment by DonJayamanne** [comment]
Once again, ignore visual studio code, just try in a terminal in side your VM and then try the command.
Forget visual studio code. Your above tests and the related information is quite confusion.

Can we start with a simple test please.

**Comment by DonJayamanne** [comment]
If something is not working in vscode, then first let's confirm it works outside vscode in this vm you are referring to.

**Comment by Scoodood** [comment]
Hi Don,

For some reason I failed to replicate the test scenario no. 2 (Win7x64 VM (within Win7x64 host). I had uninstall/reinstall/reboot a few time, uninstall other unused windows program, and somehow certain combination of those cure the problem. I should have slow down my steps because I failed to pin point the exact root cause this time. Sorry about that.

Right now everything is working fine on all environment (within vscode's run cell, within vscode's integrated terminal, outside vscode -> jupyter notebook, iPython shell within Win7 command prompt).  

I will continue to monitor its behavior and report the issue to you when it ever comes back.
Cheers!
